### DISCUSSION TITLE
印出聖誕樹

### DISCUSSION THREAD

**Discussion by penciloil** [Lidemy/forum#242] [其他問題]
![image](https://user-images.githubusercontent.com/89691317/143469804-78d71ca8-a57f-4660-9505-72ca0305af36.png)
想請問為什麼樹幹的部分顯示怪怪的呢?

**Comment by aszx87410** [comment]
看起來你的 console 自動把一樣的內容顯示在一起了，所以才會出現那個 5

**Reply by chchachen** [reply]
老師，請問console自動把一樣的內容顯示在一起是什麼意思呢？我應該如何避免這個情況發生？

**Reply by aszx87410** [reply]
就是他是出現 5 *
代表他會把同樣的內容結合在一起，只輸出一次加上次數
我也不太確定該怎麼解決，可能問問看 chatgpt?

**Reply by chchachen** [reply]
謝謝老師解說，不好意思挖出這麼久的問題出來問。
